tests/testthat/_snaps/snapshot-serialize.md

```diff
@@ -0,0 +1,11 @@
+# snapshot output doesn't use ANSI escapes
+
+    Code
+      cli::cli_alert_success("Success!")
+    Message <cliMessage>
+      v Success!
+
+---
+
```

Comment:
Why does `expect_snapshot_output()` produce nothing here?

Reply:
It's a message not output? Perhaps `expect_snapshot_output()` is confusingly named?

Reply:
I guess I had different expectations based on this in the docs:

> `expect_snapshot_output()` captures just output printed to the console